learn/getting_started/getting_ready_for_production.md

```diff
@@ -33,6 +33,7 @@ Meilisearch allows you to restrict access to API endpoints. You can protect your
 
 ::: tab CLI
 
+```bash
 ./meilisearch --master-key="your_master_key"
 
 :::
```

Comment:
You have the right idea, but you should close your code blocks as shown below:

````suggestion
::: tab CLI

```bash
./meilisearch --master-key="your_master_key"
```

:::
````

Same comment applies to two other instances below.